packages/nouns-webapp/src/pages/Vote/Vote.module.css

```diff
@@ -71,6 +71,33 @@
   border-top: 0px;
 }
 
+.desturctiveTransitionStateButton {
+  height: 50px;
+  border-radius: 8px;
+  border-color: var(--brand-color-red);
+  font-family: 'PT Root UI';
+  font-weight: bold;
+  font-size: 24px;
+  transition: all 0.125s ease-in-out;
+  background-color: var(--brand-color-red);
+}
+
+.desturctiveTransitionStateButton:hover {
+  background-color: var(--brand-color-red);
+  opacity: 0.5;
+  cursor: pointer;
+}
+
+.desturctiveTransitionStateButtonSection {
```

Comment:
`desturctive` -> `destructive`

Reply:
corrected in commit
https://github.com/EricDevito/nouns-monorepo/commit/add31e1d8bf70e4f2b9f0a5b820d60fa3e0a0d31
https://github.com/EricDevito/nouns-monorepo/commit/493711d4c9607714c3f3d538f457cd38a7bb169e